h3i/src/prompts/h3/mod.rs

```diff
@@ -423,6 +429,23 @@ fn prompt_extension() -> InquireResult<Action> {
     Ok(action)
 }
 
+pub fn prompt_connection_close() -> InquireResult<Action> {
+    let is_app = prompt_yes_no("application close:")?;
+    let error_code = prompt_varint("error code:")?;
+    let reason = Text::new("reason:")
+        .with_placeholder("empty")
```

Comment:
The field should be `reason phrase:` and the placeholder can say something like <optional reason phrase>.